rancher/labels/index.md

```diff
@@ -24,6 +26,30 @@ Key | Value |Description
 
 > **Note:** For the labels prefixed with `io.rancher.scheduler.affinity`, there are slight variations based on your how want to match (i.e. equal or not equal, hard or soft rules). More details can be found [here]({{site.baseurl}}/rancher/rancher-compose/scheduling/#table-of-scheduling-labels).
 
+### Selector Format 
+
+The values for selector labels (i.e. `io.rancher.service.selector.link`, `io.rancher.service.selector.container`) can be in multiple formats to be able to select services/containers based on their labels. 
+
+```
+# Label could be key=value1 OR key=value2
+key in (value1, value2)
+# Label cannot be key=value1 OR key=value2
+key notin (value1, value2)
+# Label is only the key (no matching value needed)
+key
+# Label must have key=value
+key = value
+# Labels must not have key=value 
+key != value
+```
+
+Any combination of the formats can be used togetehr in a comma separated list. 
+
+```
+key != value, key in (value1, value2), key=value3
+```
+
+> **Note:** If there is a label with a value that contains a comma in it, the selector will not be able to match with the label due to the ability to select on `key` alone. Example:  `io.rancher.service.selector.link: foo=bar1,bar2` would translate to any containers that have labels, key of `foo` with value `bar1` or key of `bar2` would join the service. It would NOT pick up a service that has `foo=bar1,bar2` as a label. 
```

Comment:
"key of `foo` with value `bar1` or key of `bar2`" - not quite correct. Comma indicates AND, not OR. Every statement separated by comma, should be valid in order to qualify for selector match.